spinflip/time_plots.py

```diff
@@ -42,31 +57,35 @@ def H_LR(self, avg_method, theta, phi):
                                for SPclass in self.spin_params_timearray])
         
 
-    def plot(self, quantity, avg_method = 'GM', theta=0, phi=0, set_title = None, savefig = False):
+    def plot(self, quantity, avg_method = 'GM', theta=0, phi=0, savefig = False):
 
         direction = [np.cos(phi)*np.sin(theta),np.sin(phi)*np.sin(theta),np.cos(theta)]
         f, ax = plt.subplots()
 
         if quantity == 'J_spatial': 
             J = self.J_spatial_flavoravg(avg_method)
             J_directional_projection = np.array([np.dot(J_at_t,direction) for J_at_t in J])
+
+            plt.semilogy(self.time_axis,J_directional_projection)
+
             plt.semilogy(range(self.nt),J_directional_projection)
```

Comment:
Should we remove the second semilogy command?

Reply:
yep